.gitignore

```diff
@@ -1,5 +1,5 @@
 build
-out

```

Comment:
I guess this file can be reverted ..

Reply:
Done (this was since the earlier script had `out.gn.1` as the output folder, but we don't need that)